q2_vizard/plugin_setup.py

```diff
@@ -54,7 +55,41 @@
                      ' plotted on the x-axis.',
         'y_measure': 'Numeric measure from the input Metadata that should be'
                      ' plotted on the y-axis.',
-        'color_by_group': 'Categorical measure from the input Metadata that'
-                          ' should be used for color-coding the scatterplot.',
+        'color_by': 'Categorical measure from the input Metadata that'
+                    ' should be used for color-coding the scatterplot.',
         'title': 'The title of the scatterplot.'}
 )
+
+
+plugin.visualizers.register_function(
+    function=lineplot,
+    inputs={},
+    parameters={
+        'metadata': Metadata,
+        'x_measure': Str,
+        'y_measure': Str,
+        'replicates': Bool,
+        'average': Str % Choices('median', 'mean'),
```

Comment:
I think you could combine these with something like:
```
replicate_method: Str % Choices('error', 'median', 'mean')
```
with a default of `error`. 
Then you don't have to worry about 2 of the (2*2+1=5) possible branches being dead-ends.

Reply:
i like it! i'll set the default to 'none' since i think that more closely reflects the default being a lack of replicate handling being chosen.